### PR TITLE
add java11

### PR DESCRIPTION
**Pull request by marsishandsome**

### CODE REVIEW

File: .ci/test.sh [hunk 1]
```diff
@@ -16,3 +16,6 @@ mvn clean test -am -pl flink/flink-1.12
 mvn clean test -am -pl flink/flink-1.13
 mvn clean test -am -pl mapreduce/mapreduce-base
 mvn clean test -am -pl prestodb
+
+export JAVA_HOME=/home/jenkins/agent/lib/jdk-11.0.12
+mvn clean compile -am -pl prestosql
```
Comment (humengyu2012) [inline .ci/test.sh:21]: shoule be `mvn test compile -am -pl prestosql ` ?

Comment (marsishandsome) [reply .ci/test.sh]: fixed

**Review by humengyu2012 — APPROVED** [review]
LGTM

File: .ci/build.groovy [hunk 2]
```diff
@@ -9,15 +9,20 @@ def call(ghprbActualCommit, ghprbPullId, ghprbPullTitle, ghprbPullLink, ghprbPul
                         rm -rf /maven/.m2/repository/*
                         rm -rf /maven/.m2/settings.xml
                         rm -rf ~/.m2/settings.xml
+                        archive_url=http://fileserver.pingcap.net/download/builds/pingcap/client-java/cache/tikv-client-java-m2-cache-latest.tar.gz
```
Comment (sunxiaoguang) [inline .ci/build.groovy:12]: Since we are downloading latest tikv-client-java, can we remove .ci/build-client-java.sh?

Comment (sunxiaoguang) [reply .ci/build.groovy:12]: LGTM

**Review by sunxiaoguang — APPROVED** [review]

File: .ci/build.groovy (hunk 2 above)
Comment (marsishandsome) [reply .ci/build.groovy:12]: `tikv-client-java-m2-cache-latest.tar.gz` is not java client. It the m2 cache.

Comment (marsishandsome) [reply .ci/build.groovy:12]: The `maven cache` may be inaccurate. It's better to compile by `.ci/build-client-java.sh`.

Comment (sunxiaoguang) [reply .ci/build.groovy:12]: Sure

**Pull request merged**